app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,32 @@
+@import "foundation_and_overrides";
+@import "card.scss";
+@import url(https://fonts.googleapis.com/css?family=Quicksand|Raleway);
+
+body {
+  background-image: url("background-photo-stars.jpg");
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+}
+
+.top-bar { background-color: transparent; }
+.top-bar-section { background-color: transparent !important;  }
+.top-bar-section li { background-color: transparent !important;  }
+.top-bar-section li a { background-color: transparent !important;  }
+
+.center-container {
+  position: absolute;
+  top:50%;
+  left:50%;
+  transform: translate(-50%, -50%);
+}
+
+.master-overlay {
+  background-color: rgba(250,250,250,.2);
```

Comment:
`.2` should be written with a leading zero as `0.2`<br>Commas in function arguments should be followed by a single space